CONTRIBUTING.rst

```diff
@@ -0,0 +1,109 @@
+======================
+Contributing to DWI_ML
+======================
+
+Welcome to DWI_ML ! We're excited you're here and want to contribute.
+
+This article documents how to contribute improvements to DWI_ML.
+
+The development process
+=======================
+
+When writing code, please pay attention to the following aspects:
+
+Coverage
+--------
+
```

Comment:
I would move the two first lines

(We use `pytest`_ to write the tests of the code.) (If you are adding code into a module that already has a test file, add additional tests into the respective file.) before the title "coverage".  And under a title "testing"

Reply:
:+1: Done in ad7ee3c.